python/tvm/relay/op/contrib/clml.py

```diff
@@ -37,7 +37,7 @@
 def clml_sdk_version():
     """Utility function to get clml version version"""
 
-    return tvm.support.libinfo().get("TVM_CLML_VERSION", 2)
+    return int(tvm.support.libinfo().get("CLML_VERSION_MAJOR", 2))
```

Comment:
Why did you change `TVM_CLML_VERSION` on `CLML_VERSION_MAJOR`?

Reply:
The version check from libinfo was broken earlier and not functional. ```TVM_CLML_VERSION``` was compiler flag and cmake option is ```CLML_VERSION_MAJOR```. We can maintain same for both.